### PR TITLE
Azure Installer: Use Correct Endpoint

### LINKED ISSUE #1613: Azure Installer: Use correct endpoint
In order for the full authentication flow to work, we have to authenticate with the correct endpoint. Since the app is created within the context of a domain, we have to authenticate with that domain's correct OAuth 2 endpoint (which usually looks like `https://login.microsoftonline.com/cbd880ba-ffc3-4c35-91c4-3ed01b7c8f33/oauth2/token?api-version=1.0`) and can be found by clicking on the 'View Endpoints' button in the configuration for a domain in the Azure Management Portal.

azure-app-1-1

### PR DESCRIPTION
**Pull request by felixrieseberg**
Closes #1613

In order for the full authentication flow to work, we have to authenticate with the correct endpoint. Since the app is created within the context of a domain, we have to authenticate with that domain's correct OAuth 2 endpoint (which usually looks like `https://login.microsoftonline.com/cbd880ba-ffc3-4c35-91c4-3ed01b7c8f33/oauth2/token?api-version=1.0`). This PR fixes the whole flow.

### CODE REVIEW

File: installer/app/src/views/modal/credentials.js.jsx [hunk 1]
```diff
@@ -185,9 +185,18 @@ var Credentials = React.createClass({
...
 			return;
 		}
+
+		var endpointNode = e.target.querySelector('[name=endpoint]');
+        var endpoint = endpointNode.value.trim();
+        if (endpoint === '') {
+            endpointNode.focus();
+            return;
+        }
+
```
Comment (jvatic) [inline installer/app/src/views/modal/credentials.js.jsx]: indentation is off

File: installer/app/src/views/azure-create-app-tutorial.js.jsx [hunk 2]
```diff
@@ -53,6 +53,18 @@ var AzureCreateAppTutorial = React.createClass({
...
+					<img src={AssetPaths["azure-app-1-1.png"]}  style={{
+						width: 659,
+						height: 372
+					}}/>
+					<label>
+						<p>Click on the applications tab and the "ENDPOINTS" button at the bottom</p>
+						<p>Copy your OAuth 2.0 Token Endpoint into the input below</p>
+						<input name="endpoint" type="text" placeholder="https://login.microsoftonline.com/{your-uid}/oauth2/token?api-version=1.0" />
+					</label>
+				</li>
```
Comment (jvatic) [inline installer/app/src/views/azure-create-app-tutorial.js.jsx]: This step should go after the application is created as the endpoints button doesn't exist if there aren't any applications.

**Comment by lmars** [comment]
@felixrieseberg the commit message prefix should be `installer:`

**Comment by felixrieseberg** [comment]
@lmars done.

**Comment by jvatic** [comment]
LGTM, Thanks!

**Pull request merged**